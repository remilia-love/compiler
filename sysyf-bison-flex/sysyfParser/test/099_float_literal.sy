//test float
float array[4] = {1.575e1,1575E-2,-2.5e-3,35E-4};
float sum(float a){
    float c = -1.4;
    float d = a + c - - array[0];
    return d;
}
int main(){
    sum(15.775);
    return 0;
}
