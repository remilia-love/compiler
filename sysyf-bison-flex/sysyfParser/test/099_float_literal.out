float array[4] = {15.75, 15.75, (-0.0025), 0.0035};
float sum(float a)
{
    float c = (-1.4);
    float d = ((a+c)-(-array[0]));
    return d;
}
int main()
{
    sum(15.775);
    return 0;
}
